test/specs/views/Card/CardContent-test.js

```diff
@@ -1,3 +1,6 @@
+import _ from 'lodash';
```

Comment:
Heads up, tests are failing due to lint:

```
/home/ubuntu/Semantic-UI-React/test/specs/views/Card/CardContent-test.js
  1:23  error  Extra semicolon  semi
```

You can run `npm run lint:fix` to fix most errors and then push.  Note that you can ignore warnings.